Makefile

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
This is possible, but adds the requirement that people have a working build environment.  My suggestion was going to be that we publish binaries as Github releases, but I had planned to refrain from suggesting that until the dust had settled and we (at least) had something worthy of calling a release.